src/main/resources/assets/opencomputers/loot/OpenOS/bin/dmesg.lua

```diff
@@ -0,0 +1,31 @@
+local event = require "event"
+local component = require "component"
+
+local interactive = io.output() == io.stdout
+local color, isPal, evt
+if interactive then
+  color, isPal = component.gpu.getForeground()
+end
+pcall(function()
+  repeat
+    evt = {event.pull()}
```

Comment:
Better to use `table.pack` here, to avoid dropping parameters after a `nil` one.